src/pages/index.js

```diff
@@ -39,96 +52,100 @@ const initialCards = [
     name: "Golden Gate Bridge",
     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/7-photo-by-griffin-wooldridge-from-pexels.jpg",
   },
-];
+];*/
 
-/*
-const modals = document.getElementsByClassName("modal");
-const pageAll = document.querySelector(".page-all");
-const modalContainer = document.querySelector(".modal__container");
-const modalContainerImg = document.querySelector(".modal__image-container");
-*/
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "c6481afd-8d2a-4407-bd56-fbcacf9b0f0c",
+    "Content-Type": "application/json",
+  },
+});
 
-//Profile Elements
-const editModalButton = document.querySelector(".profile__edit-btn");
-const profileName = document.querySelector(".profile__name");
-const profileDescription = document.querySelector(".profile__description");
+api.getAppInfo().then(([userInfo, cards]) => {
+  /*console.log("Full userInfo object:", userInfo);
+  console.log("Avatar URL specifically:", userInfo.avatar);
+
+  if (!Array.isArray(cards)) {
+    console.log("Cards is not an array:", typeof cards);
+    return;
+  }*/
+  cards.forEach((item) => {
+    const cardElement = getCardElement(item);
+    cardsList.prepend(cardElement);
+  });
+
+  profileAvatar.src = userInfo.avatar;
+  console.log("Avatar src after update:", profileAvatar.src);
+  profileName.textContent = userInfo.name;
+  profileDescription.textContent = userInfo.about;
+});
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere in the code**